DbAdapterTestUtilities/conanfile.py

```diff
@@ -12,7 +12,7 @@ class DbAdapterTestUtilitiesConan(ConanFile):
     license = "MIT"
     generators = "cmake_find_package"
     settings = "os", "compiler", "build_type", "arch"
-    options = {"boost": ["1.66.0", "1.67.0", "1.72.0", "1.75.0"], "gtest": ["1.7.0", "1.8.1", "1.10.0"]}
+    options = {"boost": ["1.66.0", "1.67.0", "1.72.0", "1.75.0", "1.76.0"], "gtest": ["1.7.0", "1.8.1", "1.10.0"]}
     default_options = {"boost":"1.72.0", "gtest": "1.10.0"}
```

Comment:
The same here. However we are using latest gtest version.

Reply:
Yes, good point, we should be using the latest version if not provided. Fixed.